frontend/src/components/HeadSail.js

```diff
@@ -70,7 +70,7 @@ justify-content: space-evenly;
 align-items: center;
 font-size: 0.9em;
 
-  div > div {
+  div > span {
```

Comment:
Maybe remove the duplication and introduce a class or styled component? Relying on component nesting for styling is not very robust.